src/integrators/aov.cpp

```diff
@@ -91,7 +91,7 @@ class AOVIntegrator final : public SamplingIntegrator<Float, Spectrum> {
 
             if (item[1] == "depth") {
                 m_aov_types.push_back(Type::Depth);
-                m_aov_names.push_back(item[0]);
+                m_aov_names.push_back(item[0] + ".T");
```

Comment:
That channel gets put into `root` as it doesn't have a `.` is that correct? Shouldn't we fix this behaviour in `Bitmap::split()` directly instead?

Reply:
I'm not sure it's a good idea to let `Bitmap::split` handle these cases. I think it would be better to make sure we don't create `Bitmap`s with an ambiguous layout altogether, and assume that in `split`.

Reply:
Could `split` output a warning in this case?

Reply:
I'm not sure that's desirable, what would happen then if we created a Multichannel bitmap with only the root node? e.g. `[T,U,V]` as channels. In this case there's no way to tell whether these channels should really be there or not.

Reply:
So if I understand correctly, the current behaviour will be that all channels without a prefix will be placed in `root`. Is that correct?

Reply:
yes.